### ISSUE TITLE
Showing no module named idx2numpy in mnist/model/softmax.py

### ISSUE THREAD

**Issue by SidrahJunaid** [lukejduncan/mnist-tensorflow-flask#1]
I installed that module but still getting that error
